Corgibytes.Freshli.Cli/Functionality/CacheContext.cs

```diff
@@ -2,26 +2,33 @@
 using System.IO;
 using Microsoft.EntityFrameworkCore;
 
-namespace Corgibytes.Freshli.Cli.Functionality
+namespace Corgibytes.Freshli.Cli.Functionality;
+
+public class CacheContext : DbContext
 {
-    public class CacheContext : DbContext
-    {
-        public static readonly DirectoryInfo DefaultCacheDir =
-            new DirectoryInfo(Environment.GetEnvironmentVariable("HOME") + "/.freshli");
+    #nullable enable
+    public static readonly DirectoryInfo DefaultCacheDir =
+        new DirectoryInfo(Environment.GetEnvironmentVariable("HOME") + "/.freshli");
 
-        public static DirectoryInfo CacheDir = DefaultCacheDir;
+    private static DirectoryInfo? s_cacheDir;
```

Comment:
This is something that I missed before.

I don't think it's a good idea to have the cache directory stored in a static instance variable. I think it's best handled as a normal instance variable, and for the value to be passed in via the construtor. I don't think it's a good idea to have global state like this that is modified from outside of the class. I've already made that adjustment in the PR that needs to be reconciled with this one. See https://github.com/corgibytes/freshli-cli/pull/85/commits/b8c0e0a8ba8b1d13d80d3bf8350af8409ad28e63.

Reply:
How do you intend for the instance to be shared to everything that needs to work with this data (or the object thereof)? Singleton (please no)?